runners/kubernetes

```diff
@@ -2,9 +2,17 @@
 # Set KUBERNETES_MASTER to the URL of your cluster example http://kubemaster:8080
 export KUBERNETES_MASTER
 NODE_SELECTOR=${NODE_SELECTOR:=""}
+PULL_POLICY=${PULL_POLICY:=""}
```

Comment:
no reason to default this to an empty string if you're going to wrap it with a `if -n`